full-stack-tests/core/src/frontend/TestUtility.ts

```diff
@@ -6,7 +6,7 @@ import { assert } from "chai";
 import { AccessToken, GuidString, Logger, ProcessDetector } from "@itwin/core-bentley";
 import { Project as ITwin } from "@itwin/projects-client";
 import { AuthorizationClient } from "@itwin/core-common";
-import { ElectronAppAuthorization } from "@itwin/electron-authorization/lib/cjs/ElectronFrontend";
+import { ElectronAuthorizationRenderer } from "@itwin/electron-authorization/lib/cjs/ElectronFrontend";
```

Comment:
@pbell97, @ColinKerr just brought this up but it's a bit confusing that we're importing from something called "ElectronFrontend" when we just renamed it to follow the Renderer process terminology of Electron.  Maybe we should rename that barrel file that you import from.

Reply:
Thats a good point, I can do the same for the backend file as well. I'll open a PR in @itwin/auth-clients to change them to `ElectronMain` and `ElectronRenderer`

Reply:
I think we should change the names as well.  So we'd import ElectronRendererAuthorization from ElectronRenderer and ElectronMainAuthorization from ElectronMain.

That way we've tied the main and renderer terminology to electron instead of throwing authorization in the middle

Reply:
@ColinKerr Adding that to the PR as well.

Reply:
Changes should be reflected in the latest commit.

Reply:
looks good.